### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by elharo**
@loosebazooka another security issue was found in commons compress

### CODE REVIEW

**Comment by loosebazooka** [comment]
I think you need to also update the .travis.yaml for java8 builds to be something like: https://github.com/GoogleContainerTools/jib/blob/master/.travis.yml#L7

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 sudo: false
 language: java
+dist: trusty
```
Comment (loosebazooka) [inline .travis.yml]: would probably prefer we only use try on jdk 8 instead of wholesale. (move to between line 8-9)

Comment (elharo) [reply .travis.yml]: done

**Comment by elharo** [comment]
Travis passed. For some reason Github didn't pick that up.

**Comment by loosebazooka** [comment]
Weird, they're optional though, and usually so external contributors can see failures. You can probably go ahead and merge this. Or you can trigger a rebuild directly on travis.

**Comment by loosebazooka** [comment]
I restarted one because of a cloud sdk installation failure. Lets see what happens.

**Pull request merged**